src/components/dashboard/WhoContent.native.js

```diff
@@ -0,0 +1,194 @@
+import React, { useCallback, useEffect } from 'react'
+import { FlatList, PermissionsAndroid } from 'react-native'
+import Contacts from 'react-native-contacts'
+import { get, orderBy, uniq } from 'lodash'
+import { isAndroid } from '../../lib/utils/platform'
+import { Section } from '../common'
+import Separator from '../common/layout/Separator'
+import InputText from '../common/form/InputText'
+import { withStyles } from '../../lib/styles'
+import normalize from '../../lib/utils/normalizeText'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import userStorage from '../../lib/gundb/UserStorage'
+import FeedContactItem from './FeedContactItem'
+
+const WhoContent = ({ styles, setContact, error, text, value, next, state, showNext, setValue }) => {
+  const [contacts, setContacts] = React.useState([])
+  const [initialList, setInitalList] = React.useState(contacts)
+  const [recentFeedItems, setRecentFeedItems] = React.useState([])
+  const [recentlyUsedList, setRecentlyUsedList] = React.useState([])
+
+  const getAllContacts = () => {
+    Contacts.getAll((err, contacts) => {
```

Comment:
you could use `es6-promisify` package and write promisied `react-native-contacts` wrapper and don't use callbacks as it's outdated thing
(as the pure promises to - now async await are everywhere)